src/main/resources/linking.yaml

```diff
@@ -1,3 +1,11 @@
+
+reportEmailAddress: support@openlattice.com
```

Comment:
Do we need this config in linker?

Reply:
I don't think it's used anywhere but I ported it over from the pre-existing conductor.yaml just in case. I can take it out if needed

Reply:
I think it's safe to remove, probably later even from the conductor.yaml and indexer.yaml files too, since we never use it. @geekbeast ?

Reply:
Removed!